pydeequ/suggestions.py

```diff
@@ -183,6 +185,12 @@ class CategoricalRangeRule(_RulesObject):
 
     @property
     def rule_jvm(self):
+        if IS_DEEQU_V2:
```

Comment:
Hmm ideally we would like to treat V2 as the default..

Reply:
Do you mean we should drop support for <2.x and remove this condition? Or do you mean just changing the condition to something like `if IS_DEEQU_V1:` to make Deequ 1.x the exception instead of the rule?

Reply:
Yes, Deequ 1.x as an exception seems to be better.